static_src/stores/service_instance_store.js

```diff
@@ -217,7 +217,8 @@ class ServiceInstanceStore extends BaseStore {
         const instance = this.get(binding.service_instance_guid);
         if (!instance) break; // TODO throw error
         const updatedInstance = Object.assign({}, instance, {
-          changing: false
+          changing: false,
+          error: false
```

Comment:
I don't like hasError because then you have to check `hasError` and then look for a separate error object, which is two separate places data is kept and two separate checks.